word_nerd/users/serializers.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import User
+import django.contrib.auth.password_validation as django_validators
+from django.core import exceptions
+from rest_framework import serializers
+
+
+class RegisterSerializer(serializers.ModelSerializer):
+    password2 = serializers.CharField(write_only=True)
+
+    class Meta:
+        model = User
+        fields = ['username', 'password', 'password2']
+
+    def validate(self, data):
+        password = data['password']
+        confirmed_password = data.pop('password2')
+
+        errors = {}
+        if password != confirmed_password:
+            raise serializers.ValidationError(f"Passwords don't match")
+        try:
```

Comment:
NIT: I would define `errors` here instead since this is where you need them. E.g.
```python
if password != confirmed_password:
    raise serializers.ValidationError("Passwords don't match")

errors = {}
try:
...
```